libs/langgraph/tests/test_pregel.py

```diff
@@ -220,18 +220,6 @@ def logic(inp: str) -> str:
     class State(TypedDict):
         hello: str
 
-    def node_a(state: State) -> State:
```

Comment:
ooc, why remove this?

Reply:
this isn't intended to raise exception, not sure why this was still here